packages/@jsii/go-runtime/jsii-experimental/runtime.go

```diff
@@ -52,8 +52,48 @@ func Create(fqn FQN, args []interface{}, interfaces []FQN, overrides []Override,
 		panic(err)
 	}
 
-	// client.objects[res.JsiiInstanceId] = &returns
-	client.objects[&returns] = res.JsiiInstanceId
+	client.objects[returns] = res.JsiiInstanceId
+}
+
+// Invoke will call a method on a jsii class instance. The response should be
+// decoded into the expected return type for the method being called.
+func Invoke(obj interface{}, method string, args []interface{}, returns interface{}) {
+	client := getClient()
+
+	// Find reference to class instance in client
+	refid, found := client.findObjectRef(obj)
+
+	if !found {
+		panic("No Object Found")
+	}
+
+	_, err := client.invoke(invokeRequest{
+		Api:    "invoke",
+		Method: method,
+		Args:   args,
+		Objref: objref{
+			JsiiInstanceId: refid,
+		},
+	})
+
+	if err != nil {
+		panic(err)
+	}
+}
+
+func SInvoke(fqn FQN, method string, args []interface{}, returns interface{}) {
```

Comment:
Couls we call this `InvokeStatic` instead? I get `go` likes terse... but I like descriptive?